styles.css

```diff
@@ -81,6 +81,239 @@ body {
+    background-color: #ebebff;
+    border-radius: 0.5rem;
+    padding: 0.375em;
+    margin: 0 0.31em 0 0;
+    font-size: 0.8125em;
+    letter-spacing: 0.03em;
+    font-weight: 500;
+  }
+
+  button {
+    color: #396df2;
+    background-color: #fff;
+    padding: 1em;
+    font-size: 1.0625em;
+    font-style: normal;
+    font-weight: 500;
+    border: 1px solid #7f8cff;
+    border-radius: 1em;
+    margin: 0.9375em 0;
+  }
+
+  button:hover {
+    cursor: pointer;
+    color: #fff;
+    background-color: #6070ff;
+    box-shadow: 0 0.5rem 1rem rgba(64, 83, 252, 0.24);
+    border-radius: 0.5em;
+  }
+
+  button:active {
+    color: #fff;
+    background-color: #2230d2;
+    border-radius: 0.5em;
+  }
+
+  i:hover {
+    cursor: pointer;
+  }
+
+  .canopy {
+    font-size: 0.8125em;
+    font-weight: 600;
+    color: rgba(52, 69, 99, 1);
+    font-style: normal;
+    margin-right: 1.785em;
+    list-style-type: none;
+  }
+
+  .backend {
+    color: rgba(122, 134, 154, 1);
+    margin-right: 1.875em;
+  }
+
+  .year {
+    color: rgba(122, 134, 154, 1);
+  }
+
+  .flexy {
+    display: flex;
+    padding: 0 0.94em 0 0;
+    align-items: baseline;
+  }
+
+  .last {
+    list-style-type: none;
+    font-size: 1.25em;
+    font-weight: 500;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .stretch {
+    width: 100%;
+    background-color: #f7f7f9;
+    padding: 0 0.625rem;
+    border-radius: 0.5em;
+    margin-bottom: 3%;
+  }
+
+  .about {
+    background-color: #fff;
+    margin-top: 5rem;
+    border-top-right-radius: 5em;
+    padding-bottom: 9.375rem;
+  }
+
+  .works {
+    background-color: #e5e5e5;
+    width: 100%;
+  }
+
+  .image1 {
+    align-self: center;
+  }
+
+  .new li {
+    list-style-type: none;
+  }
+
+  .fa {
+    float: right;
+  }
+
+  .rule {
+    border: 1px solid #dfe1e6;
+    margin: 1.25em 0;
+  }
+
+  .list {
+    color: #000;
+    font-size: 1.25rem;
+    font-weight: 500;
+  }
+
+  main {
+    margin-top: 6.25rem;
+  }
+
+  .main button {
+    width: 9.6875rem;
+    border-radius: 0.5rem;
+    padding: 0.75rem 0;
+  }
+
+  .main h1 {
+    font-size: 2.5rem;
+    font-weight: 700;
+    margin-bottom: 0.125rem;
+    color: #172b4d;
+  }
+
+  .main {
+    display: flex;
+    flex-direction: column;
+    padding: 5em 1.5em 0;
+  }
+
+  .main .caps {
+    color: #7f8cff;
+    font-size: 1em;
+    line-height: 1.5em;
+    font-weight: 500;
+    font-style: normal;
+    margin-top: 0;
+    margin-bottom: 1.625rem;
+  }
+
+  .main p {
+    font-size: 1em;
+    color: #344563;
+    font-weight: 400;
+    line-height: 1.5em;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .icons li {
+    list-style-type: none;
+  }
+
+  .icons {
+    display: flex;
+  }
+
+  .drop {
+    margin-top: 1.5rem;
+  }
+
+  .drop li {
+    list-style-type: none;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  .icons img {
+    width: 1.25em;
+    height: 1.19em;
+    margin-right: 1.25em;
+  }
 }
```

Comment:
Kindly implement these changes for the works section :

1. The `Languages`, `Framework` and `skills` should be in  a bolder font style as seen in the design given
2. the `lanhguages` are lsited ie we jhave their names and the icons as well but in your design we have only the icons, please fix this and resubmit 😄 

Figma design :
![image](https://github.com/BenMKT/myPortfolio/assets/20537712/dcfeede3-5b21-4f25-8dae-42ad184889db)

Your design :

![image](https://github.com/BenMKT/myPortfolio/assets/20537712/a2aff4a0-bd43-4fcb-bf2e-1b932238c47c)

Reply:
- Added `font-weight` property to the `list` class value to conform to Figma design.
- Added names to the language icons with their relevant styles to conform to the Figma design.